articles/api-auth/dynamic-client-registration.md

```diff
@@ -11,11 +11,11 @@ useCase:
   - call-api
 ---
 
-# Dynamic Client Registration
+# Dynamic Application Registration
```

Comment:
Hey @esarafianou, most changes in this doc make sense, but are you sure that we want to change the `Dynamic Client Registration` to `Dynamic Application Registration`? This is how this functionality is called in the OAuth 2.0 specification (see https://tools.ietf.org/html/rfc7591) so even though we renamed Clients with Applications in our Dashboard and terminology, this specific change looks wrong to me. Wdyt?

Reply:
Is there a consensus on which to do? I agree that we should be consistent throughout, either way.

Reply:
Both seem valid to me. `Client` refers to OIDC and `Application` to our terminology. My intention was being consistent. We can use `Client` if we prefer using the OIDC here.

Reply:
Shall I rename it to `Client` or leave it as is?

Reply:
Let's say `Client` and we can change the sidebar as well.

Reply:
I updated the PR.